backend/prompt_studio/prompt_profile_manager/profile_manager_helper.py

```diff
@@ -0,0 +1,15 @@
+from prompt_studio.prompt_profile_manager.models import ProfileManager
+
+
+class ProfileManagerHelper:
+
+    @classmethod
+    def get_profile_manager(cls, profile_manager_id):
```

Comment:
I hope pre-commit fixes got missed. Can we add typing to profile_manager_id? And also to any other related places

Reply:
Yeah, I'll make the required changes.